rebasehelper/constants.py

```diff
@@ -51,3 +51,25 @@
 
 DEFENC = locale.getpreferredencoding()
 DEFENC = 'utf-8' if DEFENC == 'ascii' else DEFENC
+
+# Taken from rpm/build/files.c
+FILES_DIRECTIVES = [
```

Comment:
Does it make sense to have `SECTION_HEADERS` in `SpecContent` and `FILES_DIRECTIVES` here?